packages/server/src/api/bannerRouter.ts

```diff
@@ -101,6 +107,25 @@ export const buildBannerRouter = (
                 variant.tickerSettings &&
                 tickerData.addTickerDataToSettings(variant.tickerSettings);
 
+            const bannerContent: BannerContent = {
+                ...variant?.bannerContent,
+            };
+            const bannerMobileContent: BannerContent = {
+                ...variant?.mobileBannerContent,
+            };
+
+            if (moduleName === LocalLanguageBannerTemplateName) {
+                const localLanguage = countryCodeToVerfiedLocalLanguage(
+                    test.name,
+                    variant.name,
+                    targeting.countryCode,
+                    { bannerHeader: variant.bannerContent?.heading },
+                );
+                bannerContent?.heading && (bannerContent.heading = localLanguage?.bannerHeader);
+                bannerMobileContent?.heading &&
+                    (bannerMobileContent.heading = localLanguage?.bannerHeader);
```

Comment:
Possible candidate for a refactor later on to set the bannerContent and bannerMobileContent properties inline with the const declaration eg after lines 111 and 114.

Reply:
Yep unsure on what refactor should look like, we tried inline in the review yesterday and it did not work. Maybe pickup again when looking into null issue below.